magma/lib/magma/server/query.rb

```diff
@@ -7,9 +7,13 @@ def action
     return failure(401, errors: ["You are unauthorized"]) unless @user && @user.can_view?(@project_name)
 
     begin
-      if @params[:query] == "::predicates"
-        return success(Magma::Predicate.to_json, "application/json")
+      case @params[:query]
+      when "::predicates"
+        return success_json(Magma::Predicate)
+      when "::model_names"
+        return success_json(answer: Magma.instance.get_project(@project_name).models.keys, format: [ 'String' ])
```

Comment:
super-nit: do you want to also provide a `"type": "Array"` here, for consistency with the other endpoints, i.e. `::attribute_names`?